src/sutta_publisher/ingester/tsv_parser.py

```diff
@@ -0,0 +1,38 @@
+from os import PathLike
+
+import pandas as pd
+from pandas._typing import ReadCsvBuffer
+
+from sutta_publisher.ingester.helper_functions import (
+    _catch_translation_en_column,
+    _reference_to_html,
+    _segment_id_to_html,
+    _split_html_on_bracket,
+    _split_ref_and_number,
+)
+from sutta_publisher.ingester.parser import BaseParser
+
+
+class TsvParser(BaseParser):
+    def __init__(self, tsv: str | PathLike[str] | ReadCsvBuffer[bytes] | ReadCsvBuffer[str]) -> None:
+        self.spreadsheet = pd.read_csv(tsv, sep="\t", header=0)
+        # columns with translations have translator's name in it (eg. translation-en-sujato) so we need regex to find English translation
+        en = _catch_translation_en_column(self.spreadsheet.columns)
+        self.spreadsheet = self.spreadsheet[[en, "html", "reference", "segment_id"]]
+        self.spreadsheet.rename(columns={en: "english"}, inplace=True)
+        self.spreadsheet.fillna("")  # replace empty cells with empty strings, so we can assume all input has str
+
+    def parse_input(self) -> str:
+        """Parse a tsv spreadsheet content and construct HTML string"""
+        # TODO: replace with Jinja2 template
```

Comment:
If you are adding todo, also add an issue for that and point to that issue in this line